src/uwtools/tests/cli/test_atparse_to_jinja2.py

```diff
@@ -26,6 +26,22 @@ def test_main_bad_args(sw, capsys):
         assert "Specify either --dry-run or --outfile" in capsys.readouterr().err
 
 
+@pytest.mark.parametrize(
+    "sw",
+    [
+        ns(d="-d", i="-i", q="-q", v="-v"),
+        ns(d="--dry-run", i="--input-template", q="--quiet", v="--verbose"),
+    ],
+)
+def test_main_bad_args_1(capsys, sw):
+    argv = ["test", sw.i, fixture_path("ww3_multi.inp.IN"), sw.d, sw.q, sw.v]
+    with patch.object(atparse_to_jinja2.sys, "argv", argv):
+        with raises(SystemExit) as e:
+            atparse_to_jinja2.main()
+        assert e.value.code == 1
+        assert "Options --quiet and --verbose may not be used together" in capsys.readouterr().err
```

Comment:
Added a test to deal with the new `--quiet` and `--verbose` flags of the `atparse-to-jinja2` program. A shame since it will disappear in UW-343, but this keeps everything running smoothly in the meantime.